docs/modules/servers/pages/distributed/index.adoc

```diff
@@ -0,0 +1,33 @@
+= James Distributed Mail Server
+:navtitle: Distributed
+
+////
+// dml: Comment for discussion (to be removed)
+ If possible, I think that the Distributed Server should be easily migratable
+ from the Extendable Server. With this in mind, I propose some updates to the
+ description. I also tried to remove "technical" elements from the overview.
+ They are necessary, but IMO should be avoided here because many different people
+ read the overview to decide if it is relevant to them or not.
+////
```

Comment:
We should have a page about such a "migration", yes, good point.

Reply:
https://github.com/apache/james-project/pull/237/commits/7731bc51bd37de1cd3fd74e1091447d47fd7aa9e proposes a migration strategy from an arbitrary IMAP server

Reply:
Note taken. Thank you!